### DISCUSSION TITLE
Messagebox styling

### DISCUSSION THREAD

**Discussion by TCExpert** [TomSchimansky/CustomTkinter#1267] [Q&A] [answered]
Hello everyone,

I made a small Calculator that lets you generate subnet adresses, subnet masks etc.. The only problem I have so far is that i havent found a way to style the messageboxes with this libary.

If it is already included it would be great if someone could tell me how and where. If not this would be a good addition to this libary.

Thank you
![IPv4 Network Calculator](https://user-images.githubusercontent.com/125686514/220552124-d52b51cd-ce79-433a-8d9b-0745d610e9eb.PNG)

**Comment by Akascape** [comment, marked as answer]
@TCExpert 
Here is a custom messagebox I made: https://pypi.org/project/CTkMessagebox/
It looks like this:
![Screenshot](https://user-images.githubusercontent.com/89206401/221258593-75058878-b598-40c3-828a-1d44a6cefb73.jpg)

**Reply by TCExpert** [reply]
@Akascape 
Thank you for the answer! Those boxes look great!

**Reply by Akascape** [reply]
@TCExpert CTkMessagebox: https://github.com/Akascape/CTkMessagebox
